packages/colorpickers/src/index.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+export * from './elements/ColorPicker';
+export * from './elements/ColorDialog';
+export * from './elements/ColorPicker/reducer';
```

Comment:
What reducer methods are we providing to consumers? Is this intended?

Reply:
Only two types definitions. Originally doing this to get rid of some HMR warnings. But updated to use named exports for specific values.